### PR TITLE
cmd/cleanup.rb: description style tweak

### PR DESCRIPTION
**Pull request by neutric**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: Library/Homebrew/cmd/cleanup.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def cleanup_args
              description: "Show what would be removed, but do not actually remove anything."
       switch "-s",
              description: "Scrub the cache, including downloads for even the latest versions. "\
-                          "Note downloads for any installed formulae or casks will still not be deleted. "\
+                          "Note: Downloads for any installed formulae or casks will still not be deleted. "\
```
Comment (EricFromCanada) [inline Library/Homebrew/cmd/cleanup.rb:29]:
```suggestion
                          "Note that downloads for any installed formulae or casks will still not be deleted. "\
```

Comment (neutric) [reply Library/Homebrew/cmd/cleanup.rb:29]: Thanks for the review, @EricFromCanada! `update-reset`, `bump-formula-pr` and the `ENVIRONMENT` section of the man page use `Note:`. Is this format reserved for cases where the note gets its own line to stand out more?

Comment (EricFromCanada) [reply Library/Homebrew/cmd/cleanup.rb:29]: Correct; a sentence starting with such a prefix should be its own paragraph.

**Review by MikeMcQuaid — APPROVED** [review]
Thanks again @neutric, good to see you back in these parts again!

**Comment by neutric** [comment]
@MikeMcQuaid Thanks for your kind words! :)

**Pull request merged**